gradle-plugin/src/test/kotlin/com/google/devtools/ksp/gradle/SourceSetConfigurationsTest.kt

```diff
@@ -72,6 +72,47 @@ class SourceSetConfigurationsTest {
         )
     }
 
+    @Test
+    fun configurationsForMultiplatformApp() {
```

Comment:
Can you please extend the test to check that final configurations "_processorClasspath" are correctly constructed? E.g having something like this in the build file:
```
configurations.filter { "processorClasspath" }.forEach {
    if (it.name = "kspJvmProcessorClasspath") it.extendsFrom == ["main"]
 }
 ```